LondonBritishLibrary/orient/BLorient747.xml

```diff
@@ -0,0 +1,341 @@
+                          </msItem>
+                          <msItem xml:id="ms_i1.20">
+                            <locus from="81r"></locus>
+                            <title type="complete" ref="LIT3309Qerellos#HomilyByEpiphanius" xml:lang="en">
+                            Homily by Epiphanius</title>
+                            <textLang mainLang="gez"></textLang>
+                          </msItem>
+                          <msItem xml:id="ms_i1.21">
+                            <locus from="83r"></locus>
+                            <title type="complete" ref="LIT3309Qerellos#SymbolByEpiphanius" xml:lang="en">
+                            Symbol of faith by Epiphanius</title>
+                            <textLang mainLang="gez"></textLang>
+                          </msItem>
+                          <msItem xml:id="ms_i1.22">
+                            <locus from="83v"></locus>
+                            <title type="complete" ref="LIT3309Qerellos#HomilyByProclus" xml:lang="en">
+                            Homily by Proclus</title>
+                            <textLang mainLang="gez"></textLang>
+                          </msItem>
+                          <msItem xml:id="ms_i1.23">
+                            <locus from="85v"></locus>
+                            <title type="complete" ref="LIT3309Qerellos#HomilyBySeverianus" xml:lang="en">
+                            Homily by Severianus</title>
+                            <textLang mainLang="gez"></textLang>
+                          </msItem>
+                          <msItem xml:id="ms_i1.24">
+                            <locus from="89r"></locus>
+                            <title type="complete" ref="LIT3309Qerellos#SymbolByGregory" xml:lang="en">
+                            Creed of Gregory Thaumsturgus</title>
+                            <textLang mainLang="gez"></textLang>
+                          </msItem>
+                          <msItem xml:id="ms_i1.25">
+                            <locus from="89v"></locus>
+                            <title type="complete" ref="LIT3309Qerellos#HomilyOnMelchizedek1" xml:lang="en">
+                            First homily by Cyril of Alexandria on Melchizedek</title>
+                            <textLang mainLang="gez"></textLang>
+                          </msItem>
+                          <msItem xml:id="ms_i1.26">
+                            <locus from="91r"></locus>
+                            <title type="complete" ref="LIT3309Qerellos#HomilyOnMelchizedek2" xml:lang="en">
+                            Second homily by Cyril of Alexandria on Melchizedek</title>
+                            <textLang mainLang="gez"></textLang>
+                          </msItem>
+                          <msItem xml:id="ms_i1.27">
+                            <locus from="92r"></locus>
+                            <title type="complete" ref="LIT3309Qerellos#TreatiseOnMelchizedek" xml:lang="en">
```

Comment:
note that for this ID LIT3309Qerellos#TreatiseOnMelchizedek the corresp in LIT3309Qerellos (LIT3332Melchiz) is missing